client/index.jsx

```diff
@@ -86,6 +90,46 @@ class Descriptions extends React.Component {
     });
   }
 
+  handlePost() {
```

Comment:
This is pretty handy, but doubt we need to add anything to the front-end - the endpoints should work without adding them to React

Reply:
Yes, I know I asked about this last week and your feeling was that we didn't need to touch the front-end, but this was my only idea for implementing CRUD, so I forged ahead.  You're probably right, but I'd like to get some clarification, which I'll do.